gitlab.go

```diff
@@ -22,24 +31,44 @@ func isGitLabURL(s string) (string, bool) {
 }
 
 // findGitLabREADME tries to find the correct README filename in a repository
-func findGitLabREADME(s string) (*source, error) {
-	u, err := url.ParseRequestURI(s)
+func findGitLabREADME(repoURL string) (*source, error) {
+	user, repo, err := userAndRepoFromURL(repoURL)
+	if err != nil {
+		return nil, err
+	}
+
+	apiRepoURL := fmt.Sprintf("https://gitlab.com/api/v4/projects/%s%s%s", user, "%2F", repo)
```

Comment:
To be safe, this should probably be escaped.

Ref: [`url.PathEscape`](https://godocs.io/net/url#PathEscape)

```suggestion
	apiRepoURL := fmt.Sprintf("https://gitlab.com/api/v4/projects/%s", url.PathEscape(user + "/" + repo))
```